client/src/components/profile-page/Profile.css

```diff
@@ -4,111 +4,139 @@
   background-color: var(--light-yellow);
 }
 
-.profile-wrapper {
+.grid-wrapper {
+  margin-top: 2em;
+  max-height: 100vh;
+  max-width: 100vw;
```

Comment:
the max-width doesn't seem to work 100%. On the mobile view the content doesn't stay in the viewport:

<img width="200" alt="Screenshot 2021-05-14 at 07 59 58" src="https://user-images.githubusercontent.com/79857938/118228510-9bf5dd00-b48a-11eb-8fdd-39fa315ee910.png">

Maybe `overflow: hidden`fixes it?!